### PR TITLE
Chore/fix missing options 10 2

### PR DESCRIPTION
**Pull request by julia-rom**
fixes[ this issue ](https://github.com/globalmangrovewatch/gmw-users/issues/361)
also combined[ this one](https://github.com/globalmangrovewatch/gmw-users/issues/356) to remove required indicator in 2.7 & 2.8 (whoops, didn't realize I was on the same branch)

- added custom checkbox group to include dropdown for paid/voluntary options in 10.2
- added findMonitoringDataItem (previously we only had it set for registration items)
- updated naming of findDataItems in other files to specify monitoring or registration

to test:
- create a site
- go to section 10
- go to 10.2
- check items and also use drop down
- save and refresh

### CODE REVIEW

**Review by mmnoo — APPROVED** [review]
Looks good. The feature works but created some loading bugs which I fixed. Left the commit for you to review and squash if you want.

**Comment by mmnoo** [comment]
I resolved merge conflicts for you to review (left a merge commit instead of rebasing).

**Comment by mmnoo** [comment]
github is also complaining that you cant rebase and merge, but it will allow squash merge and merge commit merge

File: mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js [hunk 1]
```diff
@@ -197,26 +197,30 @@ const EcologicalStatusAndOutcomesForm = () => {
...
-      axios
-        .get(monitoringFormSingularUrl)
-        .then((monitoringResponse) => {
-          const ecologicalMonitoringStakeholdersInitialVal =
-            getEcologicalMonitoringStakeholders(monitoringResponse)
-
-          const initialEcologicalMonitoringStakeholdersTypesChecked =
-            ecologicalMonitoringStakeholdersInitialVal?.map(
-              (stakeholder) => stakeholder.stakeholder
+      if (isEditMode) {
```
Comment (julia-rom) [inline mrtt-ui/src/components/EcologicalStatusAndOutcomes/EcologicalStatusAndOutcomesForm.js:203]: ah ok! missed the loading issue before - good to know about `isEditMode`

**Pull request merged**